includes/class-form-hooks.php

```
@@ -523,80 +509,61 @@ public function setStripePaymentParameters($parameters) {
-            if (!class_exists('MarchePriceCalculator')) {
-                return $intent_params;
-            }
+        // 管理者設定の取得
+        $adminNoPayment = $this->getAdminSetting('admin_no_payment', '0');
+        $isAdmin = current_user_can('administrator');
 
-            $priceCalculator = new MarchePriceCalculator();
+        // [conditional_button]タグの処理
+        $form = preg_replace_callback(
+            '/\[conditional_button\]/',
+            function($matches) use ($adminNoPayment, $isAdmin) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$matches'. (Unused Code Rules)

(UnusedFormalParameter)